app/app.rb

```diff
@@ -0,0 +1,78 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'data_mapper_setup'
+require './app/helpers/user.rb'
+
+class Chitter < Sinatra::Base
+include Helpers
+use Rack::MethodOverride
+register Sinatra::Flash
+enable :sessions
+set :session_secret, 'super secret'
+
+  get '/' do
+    erb :'home/index'
+  end
+
+  get '/user/new' do
+    @user = User.new
+    erb :'user/new', :layout => false
+  end
+
+  post '/user' do
+    @user = User.create(username: params[:username],
+                        email: params[:email],
+                        password: params[:password],
+                        password_confirmation: params[:password_confirmation])
+    if @user.save
+      session[:user_id] = @user.id
+      session[:username] = params[:username]
+      redirect('/peep')
+    else
+      flash.now[:errors] = @user.errors.full_messages
+      erb :'user/new'
+    end
+  end
+
+  get '/peep' do
+    @peeps = Peep.all
+    erb :'peep/index'
+  end
+
+  get '/peep/new' do
+    erb :'peep/new'
+  end
+
+  post '/peep' do
+    t = Time.now
+    peep = Peep.new(username: session[:username],
+                    content: params[:content],
+                    time: t.strftime("%b %e, %l:%M %p"))
+    peep.save
+    redirect('/peep')
+  end
+
+  get'/sessions/new' do
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.